docs/tutorial/image-building-best-practices/index.md

```diff
@@ -15,7 +15,7 @@ command, you can see the command that was used to create each layer within an im
 
     ```plaintext
     IMAGE               CREATED             CREATED BY                                      SIZE                COMMENT
-    a78a40cbf866        18 seconds ago      /bin/sh -c #(nop)  CMD ["node" "/app/src/ind…   0B                  
+    a78a40cbf866        18 seconds ago      /bin/sh -c #(nop)  CMD ["node" "/src/index.…    0B                  
```

Comment:
Minor nit. All `/src/index.js` must be `src/index.js` without the leading slash.